runtime/onert/core/include/util/OnertException.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_UTIL_ONERTEXCEPTION_H__
+#define __ONERT_UTIL_ONERTEXCEPTION_H__
+
+#include <iostream>
+
+namespace onert
+{
+namespace util
+{
+namespace exception
```

Comment:
These days I think we are trying to reduce namespace nesting. What about just `onert::util`? 

~~Even `onert` looks good to me, as they are exception classes.~~ Hmm.. but it breaks consistency of the namespace rule.

Reply:
LGTM other than it.

Reply:
Remove namespace exception.